pom.xml

```diff
@@ -97,16 +97,18 @@
 
     <repositories>
         <repository>
-            <id>ossrh</id>
-            <name>OSSRH Snapshots</name>
-            <url>https://oss.sonatype.org/content/repositories/snapshots</url>
+            <id>central</id>
+            <name>Central Repository</name>
+            <url>https://repo.maven.apache.org/maven2</url>
             <layout>default</layout>
             <snapshots>
-                <enabled>true</enabled>
-            </snapshots>
-            <releases>
                 <enabled>false</enabled>
-            </releases>
+            </snapshots>
+        </repository>
+        <repository>
+            <id>ion</id>
+            <name>Ion Repository</name>
+            <url>http://nexus.phx.connexta.com:8081/nexus/content/groups/ion/</url>
```

Comment:
It looks like there's no `0.0.1` version for the `com.connexta.transformation:transformation-api-java` and `com.connexta.transformation:transformation-api-rest-models` dependencies at that URL. I found them located here: http://nexus.phx.connexta.com:8081/nexus/content/repositories/ion-releases/com/connexta/transformation/.

Reply:
It looks like @oconnormi fixed the issue on the Jenkins side.